test/tests/interfacekernels/InterfaceSorption/interface_sorption_transient.i

```diff
@@ -0,0 +1,251 @@
+# This test is to verify the implementation of InterfaceSorption and its AD counterpart in transient conditions.
+# It contains two 2D blocks separated by a continuous interface.
```

Comment:
Oh no, this comment is old, sorry. Both inputs are for 1D tests since the 2D tests do not bring any additional value. I started with 2D and then realized it was unnecessary.

Reply:
Can you fix this up in the original file as well?

Reply:
I also applied the suggestion about distributed flag.